src/main/java/io/nats/client/impl/ListRequestEngine.java

```diff
@@ -72,11 +71,11 @@ int nextOffset() {
         return lastOffset + limit;
     }
 
-    List<String> getObjectList(String objectName) {
-        return JsonUtils.getObjectList(objectName, json);
-    }
-
-    List<String> getStringList(String objectName) {
-        return JsonUtils.getStringList(objectName, json);
-    }
+//    List<String> getObjectList(String objectName) {
```

Comment:
This has been commented out, can it be removed?

Reply:
Yes, removing